<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blokus Game</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/app.qrc">
    <normaloff>:/icons/icons/blokus.png</normaloff>:/icons/icons/blokus.png</iconset>
  </property>
  <widget class="QWidget" name="m_centralwidget">
   <layout class="QVBoxLayout" name="m_mainLayout"/>
  </widget>
  <widget class="QMenuBar" name="m_menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="m_helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="m_actionAbout"/>
    <addaction name="m_actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="m_languagesMenu">
    <property name="title">
     <string>&amp;Language</string>
    </property>
    <addaction name="m_actionSystemLocale"/>
    <addaction name="separator"/>
    <addaction name="m_actionEnglish"/>
    <addaction name="m_actionFrench"/>
   </widget>
   <widget class="QMenu" name="m_gameMenu">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <addaction name="m_actionQuit"/>
   </widget>
   <widget class="QMenu" name="m_themeMenu">
    <property name="title">
     <string>&amp;Theme</string>
    </property>
    <addaction name="m_actionDarkTheme"/>
    <addaction name="m_actionLightTheme"/>
   </widget>
   <addaction name="m_gameMenu"/>
   <addaction name="m_languagesMenu"/>
   <addaction name="m_themeMenu"/>
   <addaction name="m_helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="m_statusBar"/>
  <action name="m_actionAbout">
   <property name="icon">
    <iconset resource="resources/app.qrc">
     <normaloff>:/icons/icons/blokus.png</normaloff>:/icons/icons/blokus.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="m_actionAboutQt">
   <property name="icon">
    <iconset resource="resources/app.qrc">
     <normaloff>:/icons/icons/qt.png</normaloff>:/icons/icons/qt.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="m_actionSystemLocale">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>System language</string>
   </property>
  </action>
  <action name="m_actionEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/app.qrc">
     <normaloff>:/icons/icons/english.png</normaloff>:/icons/icons/english.png</iconset>
   </property>
   <property name="text">
    <string>&amp;English</string>
   </property>
  </action>
  <action name="m_actionFrench">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/app.qrc">
     <normaloff>:/icons/icons/french.png</normaloff>:/icons/icons/french.png</iconset>
   </property>
   <property name="text">
    <string>&amp;French</string>
   </property>
  </action>
  <action name="m_actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="m_actionDarkTheme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Dark</string>
   </property>
  </action>
  <action name="m_actionLightTheme">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Light</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/app.qrc"/>
 </resources>
 <connections/>
</ui>
